### ISSUE TITLE
SlevomatCodingStandard.Classes.PropertySpacing.IncorrectCountOfBlankLinesAfterProperty thinks comments are blank lines.

### ISSUE THREAD

**Issue by mikebronner** [slevomat/coding-standard#1607]
Consider the following code:
```php
    // phpcs:ignore SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingAnyTypeHint
    protected $casts = [
        "roles" => AsCollection::class,
        "uuid" => "string",
    ];
    // phpcs:ignore SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingAnyTypeHint
    protected $fillable = [
        "roles" => AsCollection::class,
        "uuid" => "string",
    ];
```

I get the following error on the `protected $casts` line:
```
Expected 0 blank lines after property, found 1.
(SlevomatCodingStandard.Classes.PropertySpacing.IncorrectCountOfBlankLinesAfterProperty)phpcs
```

My rule configuration is as follows:
```
    <rule ref="SlevomatCodingStandard.Classes.PropertySpacing">
        <properties>
            <property name="minLinesCountBeforeWithComment" type="integer" value="0" />
            <property name="maxLinesCountBeforeWithComment" type="integer" value="0" />
            <property name="minLinesCountBeforeWithoutComment" type="integer" value="0" />
            <property name="maxLinesCountBeforeWithoutComment" type="integer" value="0" />
        </properties>
    </rule>
```

My interpretation of the `WithComment` settings is that if a property has a docblock/comment, we can specify if we want empty lines between the comment and the previous property.

Is there something I am doing wrong? Thank you for your help!

**Comment by kukulich** [comment]
The problem here is that `//` is not detected as part of the property. You should change it to `/**`.
